application/controllers/QuestionAdministrationController.php

```diff
@@ -263,6 +264,32 @@ public function renderFormAux(Question $question)
         );
     }
 
+    public function actionAjaxLoadExtraOptions($questionId)
+    {
+        $questionId = (int) $questionId;
+        $question = Question::model()->findByPk($questionId);
+        if (empty($question)) {
+            throw new CHttpException(404, gT('Invalid question id'));
+        }
+
+        if (!Permission::model()->hasSurveyPermission($question->sid, 'surveycontent', 'update')) {
```

Comment:
`'surveycontent', 'read'` ? We don't save here.
In 3.X : if i don't make error : an user with read can see all form, but not submit.